sdk/halo-framework/src/main/java/com/mobgen/halo/android/framework/common/helpers/logger/Halog.java

```diff
@@ -18,11 +33,31 @@
     private static boolean mPrintDebug;
 
     /**
+     * Determines if the print mode is enabled
+     */
+    private static int mPrintLogToFilePolicy = PrintLog.NO_FILE_POLICY;
```

Comment:
Isn't it better having an object strategy that is in charge of printing instead of a flag?? This way it would be more flexible and the user would be able, for example, to send their log using network requests to a server (just an idea).